torch/_inductor/ir.py

```diff
@@ -7948,8 +7948,8 @@ def compute_size(new_size):
             inputs=inputs,
             outputs=outputs,
             constant_args=[tag, ranks, group_size],
-            output_split_sizes=output_split_sizes,
-            input_split_sizes=input_split_sizes,
+            output_split_sizes=[V.graph.sizevars.simplify(s) for s in output_split_sizes],
```

Comment:
Curious if it's possible to also perform this in lowering.py? Nothing wrong with this approach. Asking because we'll also need to update this for the the new collective IRs which are kernel agnostic.

FYI: I recently landed https://github.com/pytorch/pytorch/pull/120370, which switched OSS to use native funcols by default. Native funcols are still gated in fbcode with a jk and I plan to roll them out in fbcode next week.

Reply:
The proper fix re https://github.com/pytorch/pytorch/issues/118385 is to unconditionally generate all symbol bindings you can't have this problem. But I don't have time to do it this week so I just want to unblock Flavio real quick.

Reply:
Good to know. Thanks for the context.